be2-scala/README.md

```diff
@@ -4,11 +4,39 @@ Proof-of-personhood, spring 2021: Scala language back-end
 [TOC]
 
 ## Running the project
+<span style="color:red;font-weight:bold">
+Make sure to open be2-scala project folder as the root of your IDE workspace.
+</span>
 
+---
 There are two main possible ways of running the project :
-* import the project using IntelliJ
-* using **sbt**, execute `sbt compile` and `sbt run`
-
+* #### Option 1: Intellij
+>1. Import the project using IntelliJ
+>2. Modify the default  Run configuration 'Server', to include the following __VM option__: <br>
+>__```-Dscala.config=src/main/scala/ch/epfl/pop/config```__
```

Comment:
why is it all quoted, i.e. prefixed with `>` ?

Reply:
For pretty formatting